packages/backend/server/src/plugins/indexer/__tests__/init-es.sh

```
@@ -0,0 +1,1269 @@
+#  ------------ elasticsearch: es ------------
+## Create a index table with a mapping
+
+# https://www.elastic.co/docs/api/doc/elasticsearch/v8/operation/operation-indices-create
+
+curl -v -X DELETE "localhost:9200/doc?pretty"
+curl -v -X PUT http://127.0.0.1:9200/doc -H "content-type: application/json" --data-binary @./packages/backend/server/src/plugins/indexer/tables/doc.json
+curl -v -X GET "http://localhost:9200/doc/_mappings?pretty"
+
+{
+  "doc" : {
```

Shellcheck (0.10.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)